services/cloudaccounts/Alibaba/alibaba.go

```diff
@@ -0,0 +1,122 @@
+package alibaba
+
+import (
+	"fmt"
+	"github.com/dome9/dome9-sdk-go/services/cloudaccounts"
+	"net/http"
+	"time"
+)
+
+type CloudAccountRequest struct {
+	Name                                   string             `json:"name,omitempty"`
+	AlibabaAccountCredentialsViewPostModel AccountCredentials `json:"alibaba_account_credentials,omitempty"`
+}
+
+type AccountCredentials struct {
+	AccessKey    string `json:"access_key,omitempty"`
+	AccessSecret string `json:"access_secret,omitempty"`
+}
+
+type AccountName struct {
+	Name string `json:"name,omitempty"`
+}
+
+type CloudAccountResponse struct {
+	ID                     string             `json:"id"`
+	Name                   string             `json:"name"`
+	CreationDate           time.Time          `json:"creation_date"`
+	AlibabaAccountId       string             `json:"alibaba_account_id"`
+	Credentials            AccountCredentials `json:"credentials"`
+	OrganizationalUnitID   string             `json:"organizationalUnitId"`
+	OrganizationalUnitPath string             `json:"organizationalUnitPath"`
+	OrganizationalUnitName string             `json:"organizationalUnitName"`
+	Vendor                 string             `json:"vendor"`
+	EntityType             string             `json:"entityType"`
+	EntitySubType          string             `json:"entitySubType"`
+	Error                  string             `json:"error,omitempty"`
+}
+
+type OrganizationalUnitId struct {
+	OrganizationalUnitId string `json:"organizational_unit_id"`
+}
+
+type ID struct {
```

Comment:
where do you use this struct?
and even you are using this struct' it seems like redundant